blog/scripts/utils.js

```diff
@@ -19,7 +19,7 @@ export const [setLibs, getLibs] = (() => {
     (prodLibs, location) => {
       libs = (() => {
         const { hostname, search } = location || window.location;
-        if (!(hostname.includes('.hlx.') || hostname.includes('local'))) return prodLibs;
+        if (!['.hlx.', '.stage.', 'local'].some((i) => hostname.includes(i))) return prodLibs;
```

Comment:
Same question as another PR, is there a good reason for going with 'local' instead of 'localhost'? I can't think of hostnames that would have issues with this approach, but also can't think of a good reason to not be specific about it.